applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -81,24 +84,24 @@ def SaveTrailingEdgeNode(self):
 
         self.te.SetValue(CPFApp.TRAILING_EDGE, True)
```

Comment:
It would be nice to name this member variable with a more self-explicative name.

Reply:
Yes you are right` self.te `is not clear enough,